docs/source/conf.py

```diff
@@ -0,0 +1,37 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "Arelle iXBRL Viewer"
```

Comment:
Arbitrary line:

Can we use the same [config as Arelle](https://github.com/Arelle/Arelle/blob/df0664275676129426a9f6543c41d7906e5479c0/docs/source/conf.py)? I'd prefer if we used markdown for all of our docs instead of reStructuredText.